### PR TITLE
home 404 fix

### PR DESCRIPTION
**Pull request by BrendanCNg**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <header>
         <div class="container">
             <ul class="au-link-list au-link-list--inline">
-                <li><a href="/">Home</a></li>
+                <li><a href="https://sukhrajghuman.github.io/sukhraj-website/">Home</a></li>
```
Comment (sukhrajghuman) [inline docs/index.html]:
```suggestion
                <li><a href="#" aria-current="page">Home</a></li>
```

Comment (sukhrajghuman) [reply docs/index.html]: I rkn that will work mate^

Comment (sukhrajghuman) [reply docs/index.html]: thoughts?

Comment (sukhrajghuman) [reply docs/index.html]: loving the verified commits though

Comment (BrendanCNg) [reply docs/index.html]: yes much more thoughtful way of doing this lel 👍

**Pull request merged**